### PR TITLE
Fix file extraction integration test issues

### PR DESCRIPTION
**Pull request by thardeck**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 677320fb4d2fa53c7990884bc610ed53ee551aac into d5126d6c5c772df1d54b3c9f95f153c998c35299 )

File: spec/integration/support/inspect_changed_managed_files_examples.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@
       # directories are also extracted and to make sure to only list the actual
       # changed sub directory or file the parent directories are filtered
       actual_managed_files = actual_managed_files_list.reject { |e|
-        actual_managed_files_list.grep(/^#{e}.+/).any?
+        actual_managed_files_list.grep(/^#{e}.+/).any? || e == "."
```
Comment (tgoettlicher) [inline spec/integration/support/inspect_changed_managed_files_examples.rb]: `e` is not a speaking name for a variable (except it's 2.71828...).

**Comment by tgoettlicher** [comment]
As you didn't introduce the variable name `e` and this change doesn't justify a refactoring, it looks good to me.

File: spec/integration/support/inspect_changed_managed_files_examples.rb (hunk 1 above)
Comment (thardeck) [inline spec/integration/support/inspect_changed_managed_files_examples.rb]: e is element and it is specified one line above

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 616f4197b3c08ba581c6e4149eb4057c18a6fd1f into d5126d6c5c772df1d54b3c9f95f153c998c35299 )

**Pull request merged**